src/compojure/api/logging.clj

```diff
@@ -0,0 +1,20 @@
+(ns compojure.api.logging
+  (:require [clojure.string :as str]))
+
+;; default to console logging
+(defn log! [level x & more]
+  (let [log (fn [level more] (println (.toUpperCase (name level)) (str/join " - " more)))]
```

Comment:
It's unnecessary to create this function again for each log! call.

" - " separator is different to how c.t.l works, I think just " " would be closer to how c.t.l works.